sda/cmd/auth/main.go

```diff
@@ -264,30 +269,32 @@ func (auth AuthHandler) elixirLogin(ctx iris.Context) *OIDCData {
 
 		return nil
 	}
-	err = auth.Config.DB.UpdateUserInfo(idStruct.User, idStruct.Profile, idStruct.Email, idStruct.EdupersonEntitlement)
+	err = auth.Config.DB.UpdateUserInfo(idStruct.User, idStruct.Fullname, idStruct.Email, idStruct.EdupersonEntitlement)
 	if err != nil {
 		log.Warn("Could not log user info.")
 	}
 
 	if auth.Config.ResignJwt {
+		log.Debugf("Resigning token for user %s", idStruct.User)
 		claims := map[string]interface{}{
 			jwt.ExpirationKey: time.Now().UTC().Add(time.Duration(auth.Config.JwtTTL) * time.Hour),
 			jwt.IssuedAtKey:   time.Now().UTC(),
 			jwt.IssuerKey:     auth.Config.JwtIssuer,
-			jwt.SubjectKey:    idStruct.Profile,
+			jwt.SubjectKey:    idStruct.User,
 		}
 		token, expDate, err := generateJwtToken(claims, auth.Config.JwtPrivateKey, auth.Config.JwtSignatureAlg)
 		if err != nil {
 			log.Errorf("error when generating token: %v", err)
 		}
-		idStruct.Token = token
-		idStruct.ExpDate = expDate
+		idStruct.ResignedToken = token
+		idStruct.ExpDateResigned = expDate
 	}
 
 	log.WithFields(log.Fields{"authType": "oidc", "user": idStruct.User}).Infof("User was authenticated")
-	s3conf := getS3ConfigMap(idStruct.Token, auth.Config.S3Inbox, idStruct.User)
+	s3confInbox := getS3ConfigMap(idStruct.ResignedToken, auth.Config.S3Inbox, idStruct.User)
+	s3confDownload := getS3ConfigMap(idStruct.RawToken, auth.Config.S3Inbox, idStruct.User)
```

Comment:
Here it would be nice for download's config file to not contain the `host_base` and `host_bucket` of the inbox. 
In the future the download module of sda-cli could use this info instead for download's base url.

This could be a new issue that would be about what we want to include/exclude in download's s3cmd.conf.

Reply:
Good point! I think we may need some discussion to decide the exact information needed for it. I created the issue https://github.com/neicnordic/sensitive-data-archive/issues/1395 to address this problem.

Reply:
> Here it would be nice for download's config file to not contain the `host_base` and `host_bucket` of the inbox. 
> In the future the download module of sda-cli could use this info instead for download's base url.
> 
> This could be a new issue that would be about what we want to include/exclude in download's s3cmd.conf. 

If the download service is 'S3' compatible it should be usable with standard S3 tools and then all relevant information should exist in the config file.

Sda-cli might not be picky about what's in the config file but s3cmd is.